WordPress/src/main/java/org/wordpress/android/ui/debug/cookies/DebugCookiesAdapter.kt

```diff
@@ -0,0 +1,60 @@
+package org.wordpress.android.ui.debug.cookies
+
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.DiffUtil.Callback
+import androidx.recyclerview.widget.RecyclerView.Adapter
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import org.wordpress.android.databinding.DebugCookieItemBinding
+import org.wordpress.android.ui.debug.cookies.DebugCookiesAdapter.DebugCookieItemViewHolder
+import org.wordpress.android.ui.utils.ListItemInteraction
+import org.wordpress.android.util.viewBinding
+
+class DebugCookiesAdapter : Adapter<DebugCookieItemViewHolder>() {
```

Comment:
Same here as part 1, List Adapter in place of Adapter

Reply:
Updated in 08f7b390956ab1e060f159cae9cd3b88b204fa8c.